nullaway/src/test/java/com/uber/nullaway/NullAwayAutoSuggestTest.java

```diff
@@ -202,4 +236,36 @@ public void suggestSuppressionOnMethodRef() throws IOException {
             "}")
         .doTest();
   }
+
+  @Test
+  public void suggestCastToNonNullMultiLine() throws IOException {
+    makeTestHelper()
+        .addInputLines(
+            "Test.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  static class Foo { @Nullable Object getObj() { return null; } }",
+            "  Object test1(Foo f) {",
+            "    return f",
+            "           // comment that should not be deleted",
```

Comment:
What happens in the `nonNullF = nullVal; // some comment` case? Just `nonNullF = castToNonNull(nullVal); // some comment`, I assume?

Is there any case you can imagine where the comment ends up inside the `castToNonNull(...)` call like here, but without a newline between the comment and `)`?

Reply:
I'll add a couple more tests.  We could also have `castToNonNull(f./* comment */getObj())`